static/js/invite.js

```diff
@@ -75,9 +75,21 @@ function submit_invitation_form() {
                     invitee_emails_errored.push(value[0]);
                 });
 
+                var non_admin_error = i18n.t('Organization administrators can reactivate deactivated users from organization settings.');
+                var admin_error = i18n.t('You can reactivate deactivated users from');
```

Comment:
I think we should just construct the full string (including the HTML) here, to handle word ordering changes.  

You can pass variables into `i18n.t` to do this cleanly, and might need to use `{{{` in the template to have it render HTML.

Reply:
I changed it @timabbott